tests/cupy_tests/core_tests/test_internal.py

```diff
@@ -206,8 +208,30 @@ def test_invalid_stop_type(self):
     {'x': 2 ** 40,     'expect': 2 ** 40},
     {'x': 2 ** 40 - 1, 'expect': 2 ** 40},
     {'x': 2 ** 40 + 1, 'expect': 2 ** 41},
+    {'x': 2 ** 40 + 1, 'expect': 2 ** 41},
```

Comment:
This set of parameters is already in the test.

Reply:
I fixed.